app/views/dances/show.html.erb

```diff
@@ -1,61 +1,67 @@
-
-  <% if @dance.author %>
-    <h3>Author</h3>
-    <p><%= @dance.author %>
-  <% end %>
-
-  <% if @dance.publication %>
-    <h3>Publication</h3>
-    <p><%= @dance.publication %></p>
-  <% end %>
-
-  <% if @dance.year %>
-    (<%= @dance.year %>)</p>
-  <% end %>
-</div>
-
-<div class="dance_directions">
-  <h3>Direction</h3>
-  <p><%= @dance.direction %></p>
-</div>
-
-<% if @dance.dance_comments %>
-  <div class="comments" id="dance_commentlist">
-    <h3>Comments</h3>
-    <%= link_to 'Create Comment', new_dance_dance_comment_path(@dance) %>
-
-    <ul>
-      <% @dance_comments.each do |dc| %>
-        <li id="comment_box">
-          <h3><%= link_to dc.comment_type.name, comment_type_path(dc.comment_type_id) %></h3>
-          <h4><%= link_to dc.user.first_name, dance_comment_path(dc.user.id) %></h4>
-          <p><%= link_to dc.comment, dance_comment_path(dc) %></p>
-
-          <% if dc.creator?(current_user) %>
-            <%= link_to "Edit", edit_dance_comment_path(dc) %>
-          <% end %>
-          <% if dc.creator?(current_user) || @dance.creator?(current_user) || @admin_user %>
-            <%= link_to 'Delete', dc, method: :delete, data: { confirm: 'Are you sure?' } %>
-          <% end %>
-        </li>
+<main class="dance_show">
+  <header>
+    <ul class="user-bar">
+      <% if @dance.creator?(current_user) || @admin_user %>
+        <li><%= link_to 'Delete', @dance, method: :delete, data: { confirm: 'Are you sure?' } %></li>
+        <li><%= link_to 'Edit', edit_dance_path(@dance) %></li>
       <% end %>
+      <li>Created by: <%= @dance.user.first_name %></li>
     </ul>
-  </div>
-<% end %>
+
+    <h2><%= @dance.title %></h2>
+  </header>
+
+  <section class="dance_info">
+
+    <div>
+      <h3>Formation:</h3>
+      <p><%= link_to @dance.formation.name, formation_path(@dance.formation) %></p>
+    </div>
+
+    <div>
+      <h3>Meter:</h3>
+      <p><%= link_to @dance.meter.name, meter_path(@dance.meter) %></p>
+    </div>
+
+    <% if @dance.author %>
+      <div>
+        <h3>Author</h3>
+        <p><%= @dance.author %></p>
+      </div>
+    <% end %>
+
+    <% if @dance.publication %>
+      <div>
+        <h3>Publication</h3>
+        <p><%= @dance.publication %>
+    <% end %>
+
+    <% if @dance.year %>
+        (<%= @dance.year %>)</p>
+      </div>
+    <% end %>
+  </section>
+
+  <section class="dance_directions">
+    <div>
+      <h3>Direction</h3>
+      <p><%= @dance.direction %></p>
+  </section>
+
+  <section class="comments">
+    <header>
+      <h3>Comments</h3>
+      <%= link_to 'Create Comment', new_dance_dance_comment_path(@dance) %>
+    </header>
+
+    <% if @dance.dance_comments %>
+      <ul class="dance_comment_list">
+        <% @dance_comments.each do |dc| %>
+          <li>
+            <%= render 'dance_comments/dance_comment_box', dc: dc %>
+          </li>
+        <% end %>
+      </ul>
+    <% end %>
+  </section>
+</main>
```

Comment:
If view files get any longer than this, it is probably time to consider moving some sections of html here into partials (not necessarily to dry up code, but to simply manage the length of files)